################################################################################
SHORT DESCRIPTION: 
################################################################################
Enable S4 state in guest virtual firmware.

################################################################################
LONG DESCRIPTION: 
################################################################################
This sets the value that must be written to the PM1a/b_CNT.SLP_TYP register
for the guest to request this power state. The PM1a_CNT block is handled by
QEMU which puts the guest into S4. See SLP_TYP_S4_V0 in qemu/hw/acpi.c.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/tools/libacpi/ssdt_s4.asl
+++ b/tools/libacpi/ssdt_s4.asl
@@ -19,8 +19,8 @@ DefinitionBlock ("SSDT_S4.aml", "SSDT",
     /* Must match piix emulation */
     Name (\_S4, Package (0x04)
     {
-        0x00,  /* PM1a_CNT.SLP_TYP */
-        0x00,  /* PM1b_CNT.SLP_TYP */
+        0x06,  /* PM1a_CNT.SLP_TYP */
+        0x06,  /* PM1b_CNT.SLP_TYP */
         0x00,  /* reserved */
         0x00   /* reserved */
     })
